src/contributing/README.md

```diff
@@ -1 +1,30 @@
 # Contributing
+
+Thank you for your interest in contributing to the Styx ecosystem! This section provides detailed guides for contributing to different components of the project.
+
+## Contribution Areas
```

Comment:
I wonder about adding a warning or note about Boutiques here and then can link it to that part of the guide / any existing doc.

Reply:
Removed boutiques mention